app/assets/javascripts/passwords.js

```diff
@@ -27,6 +27,11 @@ $(document).ready(function() {
     }, 2000);
     e.clearSelection();
   });
+  
```

Comment:
Looks great to me, thankyou for adding this in.
I recommend a merge ASAP, but I don't have admin rights.

I accidentally messaged a password, instead of the intended link.  I suspect many other people are doing the same thing, which then defeats the purpose of this site